### ISSUE TITLE
Count notes with links in properties (frontmatter) as "not orphaned"

### ISSUE THREAD

**Issue by svenluijten** [Vinzent03/find-unlinked-files#58]
As far as I can tell, if I don't have a link in the _body_ of a note, it will be counted as "orphaned". Now, Obsidian has introduced the concept of "properties" (which is just rebranded yaml frontmatter), in which you can also have links like so:

```md
---
company: "[[Company A]]"
---

This is a meeting with some people from this company.
```

The above note will still be counted as "orphaned", even though it has a link to `Company A`.

**Comment by schobernoise** [comment]
I would also like to request this feature - almost all of my linked notes contain some form of frontmatter links.

**Comment by Vinzent03** [comment]
This is now resolved with bb1b21b7725b7bc3f8ad1cc2817fcfde9002f15e and released :tada:
